modules/fedimint-ln/src/lib.rs

```diff
@@ -992,35 +991,53 @@ impl ServerModule for Lightning {
         vec![
             api_endpoint! {
                 "/account",
-                async |module: &Lightning, dbtx, contract_id: ContractId| -> ContractAccount {
-                    module
-                        .get_contract_account(dbtx, contract_id)
-                        .await
-                        .ok_or_else(|| ApiError::not_found(String::from("Contract not found")))
+                async |module: &Lightning, db, contract_id: ContractId| -> Option<ContractAccount> {
+                    let mut dbtx = db.begin_transaction().await;
```

Comment:
Hmm I thought we wanted to limit the ability of modules to be able to access the raw database. Where are these transactions committed now? Today I don't think modules need to worry about committing at all, `fedimint-core` and `fedimint-server` handles that.

Reply:
Is this database already isolated?